libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -1112,6 +1112,48 @@ def RemoveExternalIps(self) -> Dict[str, str]:
     # Return the deleted external IP address for future use
     return external_ip_addresses
 
+  def AbandonFromMIG(self, instance_group: str) -> None:
+    """Abandons the instance from the managed instance group.
+
+    Args:
+      instance_group (str): The instance group that this instance should
+          be abondoned from.
+
+    Raises:
+      errors.OperationFailedError: If the request did not succeed.
+    """
+
+    def RaiseException(exception: Exception, details: str) -> None:
```

Comment:
Suggestion:

I see you're calling this method in two ways:
- `RaiseException(exception, exception.error_details)`
- `RaiseException(exception, str(exception))`

It looks like it would be enough to have a single argument in `RaiseException`: `RaiseException(exception: Exception)`

You can then pass the exception to the formatted string (you can use `{2!s}` to cast `exception` to a string, or pass `exception.error_details` if that's already a string).